ReactAndroid/src/main/java/com/facebook/react/views/text/TextAttributeProps.java

```diff
@@ -602,10 +612,34 @@ private void setTextTransform(@Nullable String textTransform) {
   private void setAccessibilityRole(@Nullable String accessibilityRole) {
     if (accessibilityRole != null) {
       mIsAccessibilityRoleSet = true;
-      mAccessibilityRole =
-          ReactAccessibilityDelegate.AccessibilityRole.fromValue(accessibilityRole);
-      mIsAccessibilityLink =
-          mAccessibilityRole.equals(ReactAccessibilityDelegate.AccessibilityRole.LINK);
+      mAccessibilityRole = AccessibilityRole.fromValue(accessibilityRole);
+      mIsAccessibilityLink = mAccessibilityRole.equals(AccessibilityRole.LINK);
+      String roleClassName =
+          AccessibilityRole.getValue(AccessibilityRole.fromValue(accessibilityRole));
+      Set<String> supportedTypes = new HashSet<String>();
```

Comment:
Do you need to create this set each time setting an accessibility role? Maybe move this out as a static set if not needed?